src/components/book.js

```diff
@@ -17,14 +18,15 @@ const Book = ({
       </div>
     </div>
     <div className="book-progress">
+      <img src={pieChart} alt="Progress Chart" width="170px" height="150px" />
```

Comment:
- [ ] I would recommend not to use inline styling. Instead, assign some class to this image and style it in a CSS file. Also, keep the progress bar a little bit smaller to look like the design and include the text `completed` below the percentage.